src/main/groovy/com/ullink/gradle/opencover/OpenCover.groovy

```diff
@@ -49,6 +55,14 @@ class OpenCover extends ConventionTask {
         openCoverExec
     }
 
+    def getReportGeneratorConsole() {
+        def reportGenerator = 'reportgenerator-'+ reportGeneratorVersion
+        Path path = Paths.get(project.gradle.gradleUserHomeDir.toString(), 'caches', 'reportgenerator', reportGenerator)
```

Comment:
Building the report generator path can be extracted to a getReportGeneratorHome() method

Reply:
Done